Kalkulator/src/calc.java

```diff
@@ -0,0 +1,81 @@
+import java.io.IOException;
+import java.util.Scanner;
+
+public class calc {
+    public static void main(String[] args) {
+        Scanner wpisz = new Scanner(System.in);

```

Comment:
Nie piszemy po polsku. Jedynie co można pisać po polsku to informacje dla użytkownika.